cinema/serializers.py

```diff
@@ -30,10 +30,25 @@ class Meta:
         fields = ("id", "name", "rows", "seats_in_row", "capacity")
 
 
+class MovieImageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ("id", "image")
+
+
 class MovieSerializer(serializers.ModelSerializer):
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id",
+            "title",
+            "description",
+            "duration",
+            "image",
```

Comment:
image field - shouldn't be here - this way image will show on all actions, but it must be only on list/retrieve as per task requirements